docs/src/understand/single-sign-on/azure/main.md

```diff
@@ -1,58 +1,76 @@
-# How to set up SSO integration with Microsoft Azure
+# How to set up SSO integration with Microsoft Entra ID
+
+## Purpous and Scope
+This document is for current and perspective users of the wire cloud, who want to manage their users with Microsoft Entra ID, in Azure.
 
 ## Preprequisites
 
-- <http://azure.microsoft.com> account, admin access to that account
-- See also {ref}`sso-generic-setup`.
+Before you begin, we are going to assume you have the following:
+- An account on <http://azure.microsoft.com>
+  * admin access to that account, sufficient to add an application, and manage users.
+- A team on https://app.wire.com/ or another wire backend, and admin access to that team.
+
+## Process
 
-## Steps
+### Creating a new application
+Go to <https://portal.azure.com/>, and login. You should be brought to the 'Microsoft Azure' home page.
+ * In the 'Azure services' section, click on 'Microsoft Entra ID'
 
-### Azure setup
+```{image} 00.png
+```
 
-Go to <https://portal.azure.com/>, and click on 'Azure Active Directory'
-in the menu to your left, then on 'Enterprise Applications':
+You should now see the 'Default Directory | Overview' page.
+ * In the menu to your left, 'Manage' should already be selected. under manage, click on 'Enterprise Applications'
 
 ```{image} 01.png
 ```
 
-Click on 'New Application':
+This should bring you to the 'Enterprise applications| All applications' page.
+ * Click on 'New Application':
 
 ```{image} 02.png
 ```
 
-Select 'Non-gallery application':
+This brings you to the 'Browse Microsoft Entra Gallery' page.
+ * Select 'Create your own application':
 
 ```{image} 03.png
 ```
 
-Fill in user-facing app name, then click 'add':
+This should have opened a 'Create your own application' window in the current page.
+ * Fill in the user-visible app name with the name for this application that you want your users to see.
+ * Leave the option selected next to 'Integrate with any other application you don\'t find in the gallery', then click 'add':
 
 ```{image} 04.png
 ```
 
-The app is now created.  If you get lost, you can always get back to
-it by selecting its name from the enterprise applications list you've
-already visited above.
+The app is now created, but is not yet configured. If you get lost, you can always get back to it by selecting its name from the 'Enterprise applications| All applications' page.
+
+### Configuring your new application
```

Comment:
Again, no title case?

Reply:
(I'm not eager to use title case... However, if you do, do it consistently :wink: )